code/game/objects/structures/crates_lockers/closets/secure/medical.dm

```diff
@@ -272,7 +272,6 @@
 	new /obj/item/sensor_device(src)
 	new /obj/item/key/ambulance(src)
 	new /obj/item/pinpointer/crew(src)
-	new /obj/item/clothing/shoes/magboots(src)

```

Comment:
I kind of feel like paramedics should keep their magboots. Saving people from an active breach is arguably even more important/more their thing than engineers.